### DISCUSSION TITLE
docs

### DISCUSSION THREAD

**Discussion by cabljac** [kaciakmaciak/react-query-subscription#42] [Ideas]
Hey! I maintain [docs.page](https://docs.page/)

If you like i could create a PR and set up a docs website. All it requires is a docs.json in the root directory, and a /docs directory with markdown. Then docs.page generates a website like this: https://melos.invertase.dev/

Later you can add custom domains etc.

**Comment by kaciakmaciak** [comment]
Hi Jacob,

Good idea. The docs.page looks really good. Atm, the project is setup with typedoc and deploys the API docs to [github pages](https://kaciakmaciak.github.io/react-query-subscription/). Once you setup the docs.page, we could change the typedoc to generate  markdown instead of HTML so it will be a part of the docs.page.

Thanks for the ideas and your contribution 🙏

**Comment by kaciakmaciak** [comment]
#44

**Comment by cabljac** [comment]
I have a typedoc CLI specifically for docs.page in dev right now, would be a cool chance to use it :)
